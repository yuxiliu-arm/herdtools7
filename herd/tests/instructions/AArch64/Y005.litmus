AArch64 Y005
(* MNEG *)

{
    0: X1 = -3;
    0: X2 = 2;
}

P0 ;
MNEG W3, W1, W2;

forall (
    0: X1 = -3 /\
    0: X2 = 2 /\
    0: X3 = 6
)
