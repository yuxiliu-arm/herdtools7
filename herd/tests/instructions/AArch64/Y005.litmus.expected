Test Y005 Required
States 1
0:X1=-3; 0:X2=2; 0:X3=6;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=-3 /\ 0:X2=2 /\ 0:X3=6)
Observation Y005 Always 1 0
Hash=f19c4b1b6a9d2c6634b0008b0c389728

